cpp/src/io/statistics/column_stats.cu

```diff
@@ -202,31 +201,28 @@ gatherIntColumnStats(stats_state_s *s, statistics_dtype dtype, uint32_t t, Stora
   vmin = shuffle(vmin);
   vmax = warp_reduce(storage.integer_stats[t / 32]).Reduce(vmax, cub::Max());
   vmax = shuffle(vmax);
-  vsum = warp_reduce(storage.integer_stats[t / 32]).Sum(vsum);
   if (!(t & 0x1f)) {
     s->warp_min[t >> 5].i_val = vmin;
     s->warp_max[t >> 5].i_val = vmax;
```

Comment:
why do we still have to do warp min/max? Is it because of has_minmax?

Reply:
Yes, that is the bottle neck.